config/config.go

```diff
@@ -70,6 +70,8 @@ func (config *Config) parseFlags() {
 	flag.StringVar(&config.Consul.Token, "consul-token", "", "The Consul ACL token")
 	flag.StringVar(&config.Consul.Tag, "consul-tag", "marathon", "Common tag name added to every service registered in Consul, should be unique for every Marathon-cluster connected to Consul")
 	flag.DurationVar(&config.Consul.Timeout, "consul-timeout", 3*time.Second, "Time limit for requests made by the Consul HTTP client. A Timeout of zero means no timeout")
+	flag.Uint32Var(&config.Consul.MaxAgentGetServicesFailures, "consul-max-agent-failures", 3, "Max count of failed responses for agent")
```

Comment:
How about:
`MaxAgentGetServicesFailures` -> `AgentFailuresTolerance`
and
"Max count of failed responses for agent" ->  "Max number of consecutive request failures for agent before removal from cache"